cdk/lib/db.ts

```diff
@@ -54,58 +54,60 @@ interface ICacclDb {
 }
 
 export abstract class CacclDbBase extends Construct implements ICacclDb {
-
   host: string;
+
   port: string;
-  dbPasswordSecret: Secret;
+
+  dbPasswordSecret: secretsmanager.Secret;
+
   clusterParameterGroupParams: { [key: string]: string } = {};
+
   instanceParameterGroupParams: { [key: string]: string } = {};
-  dbCluster: DocDbDatabaseCluster | RdsDatabaseCluster;
+
+  dbCluster: docdb.DatabaseCluster | rds.DatabaseCluster;
```

Comment:
In general, I think these classes have lots of properties and it might be nice to have some comments to explain what each represents.